py/stream.c

```diff
@@ -538,7 +538,7 @@ ssize_t mp_stream_posix_read(mp_obj_t stream, void *buf, size_t len) {
     }
 }
 
-off_t mp_stream_posix_lseek(mp_obj_t stream, off_t offset, int whence) {
+mp_off_t mp_stream_posix_lseek(mp_obj_t stream, mp_off_t offset, int whence) {
```

Comment:
I understand the motivation here, but I don't think this is a correct change.  The signature of this function should match the POSIX lseek function (apart from the first arg...).  Even if `mp_off_t != off_t` it should still match so it can be used as an adaptor for some caller who expects the POSIX signature.

Reply:
Ok I understand, updated.